build/errors.go

```diff
@@ -24,6 +24,16 @@ func ComposeErrors(errs ...error) error {
 		return nil
 	}
 
+	// If there is only a single error return it directly. This still allows
+	// callers of functions to check for specific errors
+	if len(errStrings) == 1 {
```

Comment:
this will have to be added to NebulousLabs/errors as well. We should make a PR that strips out build/errors.go.

Reply:
Maybe we should create an issue for that to make sure we don't forget about it.